scripts/discovery/diff_analyzer.py

```
@@ -0,0 +1,160 @@
+            method = entry["method"].upper()
+            traffic_set.add((norm, method))
+            path_methods[norm].add(method)
+
+            # Track query params
+            qs = parse_qs(urlparse(entry["path"]).query)
+            for key in qs:
+                path_params[(norm, method)].add(key)
+    return traffic_set, path_methods, path_params
+
+
+def build_report(spec_set, traffic_set, path_methods, path_params, spec):
```

Ruff (0.15.15)

[warning] Unused function argument: `path_methods`

(ARG001)